internal/local/dependencies.go

```diff
@@ -38,3 +39,27 @@ func FindAppDependencies(path string) (Dependencies, error) {
 
 	return Dependencies{rootDir, archivePath}, nil
 }
+
+// FindPackageJSON finds the Realm app dependencies as a package.json file
+func FindPackageJSON(path string) (Dependencies, error) {
+	app, appOK, appErr := FindApp(path)
+	if appErr != nil {
+		return Dependencies{}, appErr
+	}
+	if !appOK {
+		return Dependencies{}, nil
+	}
+
+	rootDir := filepath.Join(app.RootDir, NameFunctions)
+
+	packageJSONPath := filepath.Join(rootDir, NamePackageJSON)
+
+	_, err := os.Stat(packageJSONPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return Dependencies{}, fmt.Errorf("package.json not found at '%s'", rootDir)
+		}
+	}
+
+	return Dependencies{rootDir, packageJSONPath}, err
```

Comment:
not a fan of both returning a possible non-nil err and a return value with actual data in it (it tends to make things a bit harder to read/understand what's going on)

can we instead:

```go
if _, err := os.Stat(packageJSONPath); err != nil {
  if os.IsNotExist(err) {
    // NOTE: we might as well use NamePackageJSON in this error string as well
    return Dependencies{}, fmt.Errorf("%s not found at '%s'", NamePackageJSON, rootDir)
  }
  return Dependencies{}, err
}
return Dependencies{rootDir, packageJSONPath}, nil
```